<launch>
 <!--argument-->
 <arg name="camera/minDeg" default="-60"/>
 <arg name="camera/maxDeg" default="60"/>
 <!--map-->
 <arg name="localMap/width/float" default="4.6"/><!--2.3  -->
 <arg name="localMap/height/float" default="12.0"/><!--6.0  -->
 <arg name="localMap/resolution" default="0.05"/>
 <!--window-->
 <arg name="window/DivisionDegree" default="3"/>
 <arg name="window/height" default="0.15"/>
 <arg name="window/width" default="0.20"/><!-- 0.15 -->
 <arg name="window/minPts" default="30"/><!-- 100 -->
 <arg name="window/baseDistance" default="3.0"/><!-- 関係ない -->
 <!-- rqt_reconfigure -->
 <arg name="classification/rqt_reconfigure" default="false"/>
 <!-- output_screen -->
 <arg name="classification/output_screen" default="false"/>
 <!--node-->
 <!--classification-->
 <group if="$(arg classification/output_screen)">
   <node pkg="potbot" name="potbot_cc" type="potbot_cc" output="screen">
     <!--camera parameter-->
     <param name="camera/minDeg" value="$(arg camera/minDeg)"/>
     <param name="camera/maxDeg" value="$(arg camera/maxDeg)"/>
     <!--map parameter-->
     <param name="localMap/width/float" value="$(arg localMap/width/float)"/>
     <param name="localMap/height/float" value="$(arg localMap/height/float)"/>
     <param name="localMap/resolution" value="$(arg localMap/resolution)"/>
     <!--window parameter-->
     <param name="window/DivisionDegree" value="$(arg window/DivisionDegree)"/>
     <param name="window/height" value="$(arg window/height)"/>
     <param name="window/width" value="$(arg window/width)"/>
     <param name="window/minPts" value="$(arg window/minPts)"/>
     <param name="window/baseDistance" value="$(arg window/baseDistance)"/>
      <!-- rqt_reconfigure -->
      <param name="classification/rqt_reconfigure" value="$(arg classification/rqt_reconfigure)"/>
   </node>
 </group>
 <group unless="$(arg classification/output_screen)">
   <node pkg="potbot" name="potbot_cc" type="potbot_cc" >
     <!--camera parameter-->
     <param name="camera/minDeg" value="$(arg camera/minDeg)"/>
     <param name="camera/maxDeg" value="$(arg camera/maxDeg)"/>
     <!--map parameter-->
     <param name="localMap/width/float" value="$(arg localMap/width/float)"/>
     <param name="localMap/height/float" value="$(arg localMap/height/float)"/>
     <param name="localMap/resolution" value="$(arg localMap/resolution)"/>
     <!--window parameter-->
     <param name="window/DivisionDegree" value="$(arg window/DivisionDegree)"/>
     <param name="window/height" value="$(arg window/height)"/>
     <param name="window/width" value="$(arg window/width)"/>
     <param name="window/minPts" value="$(arg window/minPts)"/>
     <param name="window/baseDistance" value="$(arg window/baseDistance)"/>
      <!-- rqt_reconfigure -->
      <param name="classification/rqt_reconfigure" value="$(arg classification/rqt_reconfigure)"/>
   </node>
 </group>
</launch>
